cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/DistributedProgramRunner.java

```diff
@@ -749,4 +767,28 @@ private void addLogHandler(TwillPreparer twillPreparer, CConfiguration cConf) {
 
     return dependencies;
   }
+
+  /**
+   * Get namespace details for the {@link TwillPreparer} configuration
+   */
+  @VisibleForTesting
+  Map<String, String> getNamespaceConfigs(String namespace, Injector injector) {
+    DefaultNamespaceStore nsStore = new DefaultNamespaceStore(injector.getInstance(TransactionRunner.class));
+    return nsStore.get(new NamespaceId(namespace)).getConfig().getConfigs();
+  }
+
+  private Injector createInjector() {
+    return Guice.createInjector(
```

Comment:
Generally don't create new guice injector. Injector should be created by the outermost class only (e.g. at the `AppFabricServiceMain`)